### ISSUE TITLE
训练自己数据集时出现问题

### ISSUE THREAD

**Issue by heavengu** [RangiLyu/nanodet#396]
发生异常: RuntimeError
split_with_sizes expects split_sizes to sum exactly to 112 (input tensor's size at dimension -1), but got split_sizes=[9, 32]
  File "/home/ghw/nanodet/nanodet/model/head/nanodet_plus_head.py", line 191, in loss
    aux_cls_preds, aux_reg_preds = aux_preds.split(
  File "/home/ghw/nanodet/nanodet/model/arch/nanodet_plus.py", line 56, in forward_train
    loss, loss_states = self.head.loss(head_out, gt_meta, aux_preds=aux_head_out)
  File "/home/ghw/nanodet/nanodet/trainer/task.py", line 77, in training_step
    preds, loss, loss_states = self.model.forward_train(batch)
  File "/home/ghw/nanodet/nanodet/trainer/task.py", line 278, in optimizer_step
    optimizer.step(closure=optimizer_closure)
  File "/home/ghw/nanodet/tools/train.py", line 132, in main
    trainer.fit(task, train_dataloader, val_dataloader)
  File "/home/ghw/nanodet/tools/train.py", line 137, in <module>
    main(args)

**Comment by fengxueem** [comment]
作者在另一个相似 issue #377 当中回复了。
“Make sure the num_classes and reg_max in aux_head config are the same as the detection head.”
翻译一下，nanodet_plus 的配置文件当中 model->head 有 num_classes 和 reg_max 字段，同时 model->aux_head 也有 num_classes 和 reg_max 字段。要保证它们一致哦～
